src/platforms/flutter/troubleshooting.mdx

```diff
@@ -23,3 +23,9 @@ Building a Flutter app for a specific ABI using the `--target-platform` argument
 ```bash
 flutter build apk --target-platform=android-arm --split-per-abi
 ```
+
+## Native Symbolication on Android
+
+For the symbolication of Native events (C/C++) on Android, follow the configuration guidelines in the [Android Native Development Kit](/platforms/android/using-ndk/) documentation.
+
+It's important that you set `android:extractNativeLibs="true"` in the `AndroidManifest.xml` or `android.bundle.enableUncompressedNativeLibs=false` in the `gradle.properties`, or you might not have a good symbolication experience.
```

Comment:
I'd move this line to replace line 29 in the upload-debug.mdx file (I've made the edit to the file), and link to that content directly rather than put it in Troubleshooting.

Troubleshooting is the page where we tell folks how to fix a problem. If the problem is that they have set it up properly in the first place, we should tell them that at set up.

Reply:
makes sense, thanks